test/bordeaux-threads-test.lisp

```diff
@@ -273,6 +273,138 @@ Distributed under the MIT license (see LICENSE file)
   (is (bt:semaphore-p (bt:make-semaphore)))
   (is (null (bt:semaphore-p (bt:make-lock)))))
 
+(test mailbox-send-receive
```

Comment:
Shouldn't this be differentiated in different `test` blocks?

Reply:
send and receive are paired, we'd have two similar blocks with `is` put in different places then

Reply:
I know what you mean.
But to be honest, if there would be a problem, a test failure, say in line 321, I'd have to look at a lot of code to see what happened before.
This is hard to follow and maintain I suppose.